models/PasswordManager.cfc

```diff
@@ -25,8 +25,8 @@ component accessors='true' {
 		var propertyFile = wirebox.getInstance( 'PropertyFile@PropertyFile' )
 			.load( seedpropertiesPath );
 		
-		setSeed( propertyFile.get( 'seed', '' ) );
-    	setAlgorithm( propertyFile.get( 'algorithm', '' ) );
+		setSeed( propertyFile.get( 'seed', '0yJ!@1$r8p0L@r1$6yJ!@1rj' ) );
```

Comment:
There is no need to default these values here.  They are defaulted at the top of this CFC.  If you want to use this CFC with CF9, you shouldn't call this method at all.